fastlane/Fastfile

```diff
@@ -114,6 +119,72 @@ lane :release_purchases_ui_flutter do |options|
   end
 end
 
+desc "Create BC7 variant release"
+lane :release_bc7 do |options|
+  # Modify files in-place for BC7 variant
+  UI.message("Creating BC7 variant of purchases_flutter in-place...")
+
+  # Apply BC7 modifications
+  apply_bc7_modifications('..', false)
+
+  # Temporarily commit changes to satisfy git clean check
+  Dir.chdir('..') do
+    sh('git', 'add', 'pubspec.yaml', 'android/build.gradle')
+    sh('git', 'commit', '-m', '[TEMPORARY] BC7 modifications for publishing')
```

Comment:
I need to commit the files with changes, otherwise it will fail pub.dev validation. This should be fine during CI, since we won't be pushing this commit. But we reset the changes at the end anyway... It might not be undone if the flutter pub publish command fails for any reason though, but this should be fine when run on CI as I mentioned.